src/SpikingNeuralNetworks.jl

```diff
@@ -1,4 +1,6 @@
-module SNN
+module SpikingNeuralNetworks
+
+const SNN = SpikingNeuralNetworks
```

Comment:
I think we could export this, if you agree that it's useful to keep the shorter alias around.